app/logger/logger_config_azure.yaml

```
@@ -25,4 +25,4 @@ root:
   level: WARNING
   handlers: [console_handler, azure_handler]
 
-disable_existing_loggers: False
+disable_existing_loggers: False
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)